src/tests/SeleniumBrowser.php

```diff
@@ -316,4 +319,62 @@ function fillParagraphBlock( int $index = 0, string $text = '' ) {
         $this->fillTextInput( $this->getElementIfItExists( 'p', true )[$index], $text );
     }
 
+    /**
+     * @return void
+     */
+    function refreshPage(): void {
+        $this->driver->navigate()->refresh();
+    }
+
+    /**
+     * @return void
+     * @throws Exception
+     */
+    function savePostDraft() {
+        $this->click( 'button[aria-label="Save draft"]' );
+        // Once saved, we are redirected to a URL with a post ID. We wait for the save to complete.
+        $condition = WebDriverExpectedCondition::urlContains( 'post=' );
+        $this->waitForCondition( $condition );
+    }
+
+    /**
+     * @return string
+     */
+    function getCurrentPostID(): string
+    {
+        $url = $this->driver->getCurrentUrl();
+        preg_match('/post=(\d+)(&|$)/', $url, $matches );
+        return $matches[1];
+    }
+
+    /**
+     * Delete posts corresponding to post IDs. This assumes the posts are all on the front
+     * page of the WP Admin Post Editor list.
+     * @return void
+     * @throws Exception
+     */
+    function deletePosts( array $postIDs ) {
+        $this->openPage( WP_POSTS_HOME );
+        # Select all the posts to move to the trash.
+        foreach ( $postIDs as $postID ) {
+            $this->click( "#cb-select-{$postID}" );
+        }
+        # Specify we want to move them to the trash.
+        $this->click('#bulk-action-selector-top' );
+        $this->click('option[value="trash"]' );
+        # Move them to the trash.
+        $this->click('#doaction' );
+        # Now we need to go to the Trash tab.
+        $this->click('li.trash' );
+        # Select all the posts to delete permanently.
+        foreach ( $postIDs as $postID ) {
+            $this->click( "#cb-select-{$postID}" );
+        }
+        # Specify we want to delete them.
+        $this->click('#bulk-action-selector-top' );
+        $this->click('option[value="delete"]' );
+        # Delete them.
+        $this->click('#doaction' );
```

Comment:
I didn't see any way to permanently delete posts without first moving them to the trash. Note that we don't have access to functions like `wp_delete_post()` because we aren't in the WordPress instance.